content/300-guides/050-database/850-using-prisma-with-planetscale.mdx

```diff
@@ -133,13 +133,15 @@ model Comment {
 
 You can then add this change to your schema [using `db push`](#how-to-make-schema-changes-with-db-push).
 
+In Prisma versions 4.7.0 and later, Prisma will warn you if you have a relation with no index on the relation scalar field. For more information, see [Index validation](/concepts/components/prisma-schema/relations/relation-mode#index-validation) in the relation mode documentation.
```

Comment:
Consider using present simple tense. Also, I think the end of the second sentence is not needed.

```suggestion
In Prisma versions 4.7.0 and later, Prisma warns you if you have a relation with no index on the relation scalar field. For more information, see [Index validation](/concepts/components/prisma-schema/relations/relation-mode#index-validation).
```